echo-library/src/common/starlark_modules.rs

```diff
@@ -131,11 +131,41 @@ pub fn starlark_workflow_module(builder: &mut GlobalsBuilder) {
     fn argument(
         name: String,
         input_type: Value,
-        default_value: Option<String>,
+        default_value: Option<Value>,
     ) -> anyhow::Result<Input> {
         let input_type: RustType = serde_json::from_str(&input_type.to_json()?)
             .map_err(|err| anyhow!("Failed to parse input arguments: {}", err))?;
 
+        let default_value: Option<String> = match default_value {
+            Some(value) => {
+                let value_str = value
+                    .to_json()
+                    .map_err(|err| anyhow!("Failed to parse default value: {}", err))?;
+
+                if input_type == RustType::Int
```

Comment:
it possible like the RustType Int with a Boolean default value. condition should be modified to check the type and value